openapi-specs/schema.json

```diff
@@ -40,11 +45,11 @@
             "type": "string"
           }
         },
-        "user_id": {
+        "userId": {
           "type": "string",
           "description": "The user id of the given user"
         },
-        "user_type":{
+        "userType":{
```

Comment:
Ah, I see that we have `userType` also defined here 🤔 Is there a way to name these to be more explicit about which "type" they are? (dynamic vs static/db vs oidc)

Reply:
ok with `dbUserType`?